### ISSUE TITLE
about multi-organ segmentation loss: Dice+Focal loss

### ISSUE THREAD

**Issue by GuobinZhangTJU** [JunMa11/SegLoss#9]
Hi, 	Jun, thank you very much for your valuable work.
When you perform multi-organ segmentation on nnUNet, you observed that the combine of Dice loss and Focal loss achieved the best DSC. Can you share your parameters used in Focal loss? Such as the alpha and gamma and learning rate.
Many thanks, waiting for your reply.

**Comment by JunMa11** [comment]
Sure!

`alpha=0.5, gamma=2, learning rate=3e-5`

**Comment by GuobinZhangTJU** [comment]
> Sure!
> 
> `alpha=0.5, gamma=2, learning rate=3e-5`

many thanks~~

**Comment by GuobinZhangTJU** [comment]
> Sure!
> 
> `alpha=0.5, gamma=2, learning rate=3e-5`

Does the initial learning rate is 3e-5 based on nnunet???

**Comment by JunMa11** [comment]
Yes.

**Comment by GuobinZhangTJU** [comment]
> Yes.

ok thanks~

**Comment by GuobinZhangTJU** [comment]
Hi, may I get some advices about another question?
I tried to use nnunet to train a multi-organ segmentation model, but the performance is not very well.
The dataset we used have a characteristic: in each case, the labeled organ 1 occured in many slices but labeld organ 2 only occured in few slices because the organ 2 is very small. So the results of organ 2 is not very well. The nnunet automatic adopt dice+CE as the loss function, can you give me some advices about loss function to solve this problem~
many thanks.

**Comment by JunMa11** [comment]
Hi @GuobinZhangTJU ,

Great question. However, I also suffer from this problem.

I think this is still an open question. Currently, to the best of my knowledge, none of the losses can perfectly solve this problem.

**Comment by GuobinZhangTJU** [comment]
> Hi @GuobinZhangTJU ,
> 
> Great question. However, I also suffer from this problem.
> 
> I think this is still an open question. Currently, to the best of my knowledge, none of the losses can perfectly solve this problem.

get it~~   thanks for your reply~
